homeassistant/components/stream/const.py

```diff
@@ -48,7 +49,7 @@
 CONF_PART_DURATION = "part_duration"
 CONF_SEGMENT_DURATION = "segment_duration"
 
-CONF_PREFER_TCP = "prefer_tcp"
+ATTR_PREFER_TCP = "prefer_tcp"
```

Comment:
Reaming this since its not a configuration option. This is not exposed in `__all__` today.